js/main.js

```diff
@@ -1,17 +1,92 @@
-const comment = document.querySelector('.social__footer-text');
-
-function getRandomNumber(minNumber, maxNumber) {
-  if (maxNumber > minNumber && maxNumber > 0 && minNumber >= 0) {
-    minNumber = Math.ceil(minNumber);
-    maxNumber = Math.floor(maxNumber);
-    return Math.floor(Math.random() * (maxNumber - minNumber + 1)) + minNumber;
-  }
-  const error = 'Введите другие числа';
-  return error;
+// Количество фото
+const MIN_AMOUNT_PHOTO = 1;
+const MAX_AMOUNT_PHOTO = 25;
+
+// Количество лайков
+const MIN_AMOUNT_LIKE = 15;
+const MAX_AMOUNT_LIKE = 200;
+
+// Количество комментариев
+const FIRST_COMMENT = 1;
+const LAST_COMMENT = 3;
+
+// Идентификатор аватарок пользователей
+const FIRST_AVATAR = 1;
+const LAST_AVATAR = 6;
```

Comment:
первый и последний аватар чего? именование неудачное
```suggestion
const MIN_AVATAR_IMAGE_NUMBER = 1;
const MAX_AVATAR_IMAGE_NUMBER = 6;
```

мин/макс - потому что в коде ты уже использовала границу, как мин/макс, а не первый/последний, а код надо писать единообразно